benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadWorker.java

```diff
@@ -137,13 +140,14 @@ public static void main(String[] args) throws Exception {
     private LoadClient workerClient;
 
     @Override
-    public StreamObserver<Control.ServerArgs> runServer(
+    public StreamObserver<ServerArgs> runServer(
         final StreamObserver<Control.ServerStatus> responseObserver) {
-      return new StreamObserver<Control.ServerArgs>() {
+      return new StreamObserver<ServerArgs>() {
         @Override
-        public void onNext(Control.ServerArgs value) {
+        public void onNext(ServerArgs value) {
           try {
-            if (value.getSetup() != null && workerServer == null) {
+            ArgtypeCase argTypeCase = value.getArgtypeCase();
+            if (argTypeCase == ServerArgs.ArgtypeCase.SETUP && workerServer == null) {
```

Comment:
I can't relate this change to the PR description. Can you provide some background?